projects/kit/pipes/stringify/stringify.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {TuiStringHandler} from '@taiga-ui/cdk';
+
+@Pipe({
+    name: 'tuiStringify',
+})
+export class TuiStringifyPipe implements PipeTransform {
+    public transform<K extends string>(
+        key: K,
+    ): TuiStringHandler<Readonly<Record<K, unknown> & Record<any, any>>> {
+        return value => String(value[key] ?? '');
```

Comment:
`??` it's work for old version typescript?

Reply:
Yes, it works with our TS.